crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -1039,7 +1043,19 @@ pub fn parse_conditions<V: SpendVisitor>(
                 *max_cost -= AGG_SIG_COST;
                 ret.condition_cost += AGG_SIG_COST;
             }
-            _ => (),
+            _ => {
+                if (flags & COST_CONDITIONS) != 0 {
```

Comment:
maybe it was more reasonable the way you had it before, where it always applied to all conditions. I suppose we'll have to update a lot of tests that way. But the `SOFTFORK` condition causes complications with this

Reply:
implemented